docs/core/project-sdk/overview.md

```diff
@@ -0,0 +1,119 @@
+
+## Available SDKs
+
+The following SDKs are available for .NET Core:
+
+| ID | Description | Repo|
+| - | - | - |
+| `Microsoft.NET.Sdk` | The .NET Core SDK | https://github.com/dotnet/sdk |
+| `Microsoft.NET.Sdk.Web` | The .NET Core [Web SDK](/aspnet/core/razor-pages/web-sdk) | https://github.com/aspnet/websdk |
+| `Microsoft.NET.Sdk.Razor` | The .NET Core [Razor SDK](/aspnet/core/razor-pages/sdk) |
+| `Microsoft.NET.Sdk.Worker` | The .NET Core Worker Service SDK |
+| `Microsoft.NET.Sdk.WindowsDesktop` | The .NET Core WinForms and WPF SDK |
+
+The .NET Core SDK is the base SDK for .NET Core. The other SDKs depend on the .NET Core SDK. For example, the Web SDK depends on both the .NET Core SDK and the Razor SDK.
+
+## Project files
+
+.NET Core projects are based on the [MSBuild](/visualstudio/msbuild/msbuild) format. Project files, which have extensions like *.csproj* for C# projects and *.fsproj* for F# projects, are in XML format. The root element of an MSBuild project file is the [Project](/msbuild/project-element-msbuild) element. The `Project` element has an optional `Sdk` attribute that specifies which SDK (and version) to use. To use the .NET Core tools and build your code, set the `Sdk` attribute to one of the IDs in the [Available SDKs](#available-sdks) table.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  ...
+</Project>
+```
+
+Another way to specify the SDK is with the top-level [Sdk](/visualstudio/msbuild/sdk-element-msbuild) element:
+
+```xml
+<Project>
+  <Sdk Name="Microsoft.NET.Sdk" />
+  ...
+</Project>
+```
+
+Referencing an SDK in one of these ways greatly simplifies project files for .NET Core. While evaluating the project, MSBuild adds implicit imports for `Sdk.props` at the top of the project file and `Sdk.targets` at the bottom.
+
+```xml
+<Project>
+  <!-- Implicit top import -->
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  ...
+  <!-- Implicit bottom import -->
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+</Project>
+```
+
+> [!TIP]
+> On a Windows machine, the *Sdk.props* and *Sdk.targets* files can be found in the *%ProgramFiles%\dotnet\sdk\\[version]\Sdks\Microsoft.NET.Sdk\Sdk* folder.
+
+### Preprocess the project file
+
+You can see the fully expanded project as MSBuild sees it after the SDK and its targets are included by using the `dotnet msbuild -preprocess` command. The [preprocess](/visualstudio/msbuild/msbuild-command-line-reference#preprocess) switch of the [`dotnet msbuild`](../tools/dotnet-msbuild.md) command shows which files are imported, their sources, and their contributions to the build without actually building the project.
+
+If the project has multiple target frameworks, focus the results of the command on only one framework by specifying it as an MSBuild property. For example:
+
+`dotnet msbuild -property:TargetFramework=netcoreapp2.0 -preprocess:output.xml`
+
+### Default compilation includes
```

Comment:
should this be removed from here then?
https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#default-compilation-includes-in-net-core-projects

Reply:
also make sure https://aka.ms/sdkimplicititems is updated when this goes live

Reply:
I had kind of decided not to touch the csproj.md file in this PR and just update it separately after this goes live. For one thing, if edits need to be made to csproj.md before this goes live, then nothing is in limbo and I can go back and verify that all the important info is captured in the new reference.

Reply:
sounds good!

Reply:
The page is live and the aka.ms link has been updated. Thanks for letting me know.